.github/workflows/release-dispatch.yml

```diff
@@ -0,0 +1,18 @@
+name: release-dispatch
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  send-release-dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.PLATFORM_GITHUB_TOKEN }}
```

Comment:
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Oof, I thought the GITHUB_TOKEN would have enough permissions, but maybe not. Investigating...